packages/atomic/src/styles/variables.pcss

```diff
@@ -0,0 +1,20 @@
+:root {
```

Comment:
I think a better name for this file would be `coveo-theme.pcss`.

Reply:
what about default-theme ?

Reply:
Also a good option 👍